CHANGELOG.md

```diff
@@ -15,6 +15,24 @@ Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `
 
 ### Fixed
 
+## [26.11.3] - 2021-10-05
```

Comment:
The pr show the diff between 26.11.0 to 26.11.3 but 26.11.2 is already merged in main. I do not understand why github display a diff for it? It is only wrong displaying for me?